File: src/test/java/fi/metatavu/metaform/server/FileUploadResponse.java
package fi.metatavu.metaform.server;

import java.util.UUID;

/**
 * Model for file upload response
 * 
 * @author Antti Leppä
 */
public class FileUploadResponse {

  private UUID fileRef;
  private String fileName;

  /**
   * @return the fileName
   */
  public String getFileName() {
    return fileName;
  }

  /**
   * @param fileName the fileName to set
   */
  public void setFileName(String fileName) {
    this.fileName = fileName;
  }

  /**
   * @return the fileRef
   */
  public UUID getFileRef() {
    return fileRef;
  }

  /**
   * @param fileRef the fileRef to set
   */
  public void setFileRef(UUID fileRef) {
    this.fileRef = fileRef;
  }

} 

File: src/test/java/fi/metatavu/metaform/server/test/functional/FileUploadResponse.kt
package fi.metatavu.metaform.server.test.functional

import com.fasterxml.jackson.annotation.JsonProperty
import java.util.*

/**
 * Model for file upload response
 *
 * @author Antti Leppä
 */
data class FileUploadResponse (
    @JsonProperty("fileRef")
    val fileRef: UUID,
    @JsonProperty("fileName")
    val fileName: String
)